src/pages/Storefront/Cart.js

```diff
@@ -0,0 +1,119 @@
+    if (checkoutState.webUrl) {
+      console.log('webUrl exists');
+    }
+    // window.open(checkoutState.webUrl) // opens checkout in a new window
+    window.location.replace(checkoutState.webUrl); // opens checkout in same window
+  };
+
+  const applyCoupon = () => {
+    alert('To be implemented');
+  };
+
+  return (
+    <>
+      <Box bg="black" color="black" px="105px">
+        <Heading alignSelf="flex-start" color="white" py={6}>
+          MY SHOPPING BAG
+        </Heading>
+      </Box>
+      <HStack w="100%" h="100%" justifyContent="space-between" alignItems="flex-start" px="105px">
+        <VStack w="100%" alignItems="flex-start" py="64px" pr="45px">
+          <Flex w="100%" justifyContent="space-between">
+            <Heading as="h4" size="subtitle" color="brand.black">
+              CONTINUE SHOPPING
+            </Heading>
+            <Heading as="h4" color="brand.gray" size="subtitle" textTransform="uppercase">
+              {/* TO DO: update this to be the length of the line items in cart */}
+              {`${products.length} ITEMS`}
+            </Heading>
+          </Flex>
+          <Divider borderColor="brand.gray" />
+          {/* TO DO: The products here should be the line-items in the Shopify Checkout, also need to add unique key */}
+          {products &&
+            products.map(product => (
+              <Box w="100%" pt="28px">
+                <CartItem key={product.id} product={product} />
+                <Divider borderColor="brand.gray" pb="36px" />
+              </Box>
+            ))}
+          <HStack py="30px">
+            <FormControl w="40%">
+              <Input
+                type="coupon"
+                name="coupon"
+                placeholder="123ABCD"
+                // onChange={e => handleInputChange(e, i)}
+              />
+            </FormControl>
+            <Button size="sm" onClick={applyCoupon}>
+              APPLY COUPON
+            </Button>
+          </HStack>
+        </VStack>
+        <VStack alignItems="flex-start" pt="92px">
+          <VStack alignItems="flex-start" bg="brand.lightgray" spacing="40px" p="36px" w="409px" mb="24px">
+            {/* The current values in Order Summary are just placeholders */}
+            <Heading as="h4" size="subtitle" color="brand.black">
+              ORDER SUMMARY
+            </Heading>
+            <VStack w="100%" alignItems="flex-start" spacing="30px">
+              <Flex w="100%" justifyContent="space-between">
+                <Text>SUBTOTAL</Text>
+                <Text fontWeight="semibold">$1545</Text>
+              </Flex>
+              <Flex w="100%" justifyContent="space-between">
+                <Text>SHIPPING ESTIMATE</Text>
+                <Text fontWeight="semibold">$5</Text>
+              </Flex>
+              <Flex w="100%" justifyContent="space-between">
+                <Text>TAXES ESTIMATE</Text>
+                <Text fontWeight="semibold">$200</Text>
+              </Flex>
```

Comment:
Nit (this is unnecessary and just a suggestion): store titles and prices in an array & map through to render

```
const orderDetails = [
    {
       title: "Subtotal",
       price: "$1545"
    },
   etc ... 
]

{orderDetails.map(( {title, price} ) => (
    <Flex w="100%" justifyContent="space-between">
        <Text textTransform="uppercase" >{title}</Text>
        <Text fontWeight="semibold">{price}</Text>
    </Flex>
)}

Reply:
yeah this might be worth doing, but let's hold off on this refactor for now until we see what the `checkout` object looks like.